src/components/molecules/message/message.scss

```
@@ -0,0 +1,36 @@
+@use "../../variables" as *;
+
+.Message{
+    font-weight: 400;
+    text-align: center;
+    padding: 50px 20px;
+
+    h1{
+        font-family: $fontfamily-mincho;
+        font-size: $fontsize-xl;
+        line-height: 54;
```

line-heightの単位なしの値の仕様として、この値に要素のフォントサイズを掛けたものがline-heightになるので、今回の場合line-heightは
54×36px=1944pxとなります。単位なしを利用する場合はこの仕様を考慮しながら使用してください。
また、単位なしと単位ありが混在している(pタグやメディアクエリの中)ので統一してください
今回の場合は以下のようになると思います。
```suggestion
        line-height: 1.5;
```